public/app/plugins/datasource/influxdb/components/editor/config-v2/AdvancedDBConnectionSettings.tsx

```diff
@@ -125,9 +126,13 @@ export const AdvancedDbConnectionSettings = (props: Props) => {
                 className="width-15"
                 data-testid="influxdb-v2-config-max-series"
                 onBlur={trackInfluxDBConfigV2AdvancedDbConnectionSettingsMaxSeriesClicked}
-                onChange={onUpdateDatasourceJsonDataOption(props, 'maxSeries')}
+                onChange={(e) => {
+                  setMaxSeriesValue(e.currentTarget.value);
```

Comment:
I think set value should be done in a function that also does the on update function